infra/src/test/kotlin/fi/fta/geoviite/infra/DBTestBase.kt

```diff
@@ -2,10 +2,29 @@ package fi.fta.geoviite.infra
 
 import currentUser
 import fi.fta.geoviite.infra.authorization.UserName
-import fi.fta.geoviite.infra.common.*
```

Comment:
Tästä eteenpäin testikoodia. Kaikki testit korjattu samalla tapaa, eli sanomalla eksplisiittisesti branchin olevan aina main:
- Jos tarvitaan branch: LayoutBranch.main
- Jos tarvitaan layoutContext: `PublicationState.DRAFT -> MainLayoutContext.draft`  ja `PublicationState.OFFICIAL` -> `MainLayoutContext.official`